service/src/main/resources/db/migration/V216__message_account_evaka_user_id.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE message_account ADD COLUMN evaka_user_id uuid UNIQUE REFERENCES evaka_user (id);
+UPDATE message_account SET evaka_user_id = coalesce(person_id, employee_id);
+UPDATE message_account SET employee_id = NULL WHERE employee_id IS NOT NULL AND NOT active;
+ALTER TABLE message_account DROP COLUMN active;
+
+ALTER TABLE message_account
+    DROP CONSTRAINT message_account_person_id_fkey,
+    ADD CONSTRAINT message_account_person_id_fkey FOREIGN KEY (person_id) REFERENCES person (id) ON DELETE SET NULL,
```

Comment:
This was `ON DELETE CASCADE`, and that was the culprit of the `InactivePeopleCleaup` failure.